lib/itamae/resource/gem_package.rb

```diff
@@ -43,6 +45,13 @@ def action_install(action_options)
         end
       end
 
+      def action_uninstall(action_options)
+        if current.installed
+          uninstall!
+          updated!
```

Comment:
> Itamae marks the resource as updated automatically.

@ryotarai Did you mean this ?
https://github.com/itamae-kitchen/itamae/blob/169debf335001160644f534cebb5007a9587cc50/lib/itamae/resource/base.rb#L203-L206